draft-ietf-ppm-dap.md

```diff
@@ -1055,19 +1054,28 @@ where `pk` is the Aggregator's public key; `0x01` represents the Role of the
 sender (always the Client); `server_role` is the Role of the intended recipient
 (`0x02` for the Leader and `0x03` for the Helper), `plaintext_input_share` is
 the Aggregator's PlaintextInputShare, and `input_share_aad` is an encoded
-message of type InputShareAad defined below, constructed from the same values as
-the corresponding fields in the report. The `SealBase()` function is as
+message of type InputShareAad defined below. The `SealBase()` function is as
 specified in {{!HPKE, Section 6.1}} for the ciphersuite indicated by the HPKE
 configuration.
 
 ~~~
+struct {
+  ReportID report_id;
+  Time time;
+} ReportMetadata;
```

Comment:
Does ReportMetadata still have meaning? would it be simpler if we simply put reportID and time in InputShareAad?

Reply:
`struct ReportMetadata` ends up being useful in the aggregation interaction (e.g. it goes into a `ReportShare`) so I think it's useful.